reana_client/reana_api.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, B       else:
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+"""REANA API client."""
+
+import requests
+
+
+class ReanaAPI(object):
+    """REANA API client code."""
+
+    def __init__(self, server_url, apipath):
+        """Initialize ReanaAPI object."""
+        self.server_url = server_url
+        self.apipath = apipath if apipath is not None else '/api'
```

Comment:
I prefer the
```
if apipath is None:
        apipath='/api'
```
with ```__init__(self, server_url, apipath=None)```
or the
```
apipath or '/api'
```
shortcut.
But I guess there was a reason for having this here for now and changing it later?